docs/maps.md

```diff
@@ -0,0 +1,142 @@
+# Maps
+
+The `Maps` class provides access to the Mapbox Maps API.  You can import it from either the `mapbox` module or the `mapbox.services.maps` module.
+
+__mapbox__:
+
+```python
+>>> from mapbox import Maps
+```
+
+__mapbox.services.maps__:
+
+```python
+>>> from mapbox.services.maps import Maps
+```
+
+See https://www.mapbox.com/api-documentation/#maps for general documentation of the API.
+
+Use of the Maps API requires an access token, which you should set in your environment.  For more information, see the [access tokens](access_tokens.md) documentation.
+
+## Maps Methods
+
+The public methods of the `Maps` class provide access to the Maps API and return an instance of [`requests.Response`](http://docs.python-requests.org/en/latest/api/#requests.Response).
+
+## Usage: Retrieving Tiles
+
+Instantiate `Maps`.
+
+```python
+>>> maps = Maps()
+```
+
+Call the `get_tile` method, passing in values for `map_id`, `z` (zoom), `x` (column), and `y` (row).  Pass in values for optional arguments as necessary - `retina` (double scale), `file_format`, `style_id`, and `timestamp`.
+
+```python
+>>> response = maps.get_tile("mapbox.streets", z=0, x=0, y=0)
```

Comment:
@critical-path @perrygeo I think I'd prefer the tile params to be positional arguments in x, y, z order so we can pass in mercantile tiles like

`get_tile(mapid, *Tile(0, 0, 0))`

"get" is implied in the method, so how about just `tile()` ?